Simplenote/Controllers/AccountDeletionController.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+class AccountDeletionController {
+    var successHandler: (() -> Void)?
+    var failureHandler: ((Int) -> Void)?
```

Comment:
Curious if there is a benefit of using two separate handlers and not send a completion directly to `requestAccountDeletion`. I feel that would be a more common pattern.

It's also hard to tell what `Int` is in `((Int) -> Void)?`.

Reply:
^ I was gonna suggest the same thing. Swift's [Result Type](https://developer.apple.com/documentation/swift/result), and a single callback, passed directly to `requestAccountDeletion`